src/Sylius/Bundle/TaxonomyBundle/DependencyInjection/Configuration.php

```diff
@@ -76,12 +79,15 @@ private function addResourcesSection(ArrayNodeDefinition $node)
                                         ->scalarNode('model')->defaultValue(Taxon::class)->cannotBeEmpty()->end()
                                         ->scalarNode('interface')->defaultValue(TaxonInterface::class)->cannotBeEmpty()->end()
                                         ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
-                                        ->scalarNode('repository')->cannotBeEmpty()->end()
+                                        ->scalarNode('repository')->defaultValue(TaxonRepository::class)->cannotBeEmpty()->end()
```

Comment:
Cause the driver is loaded before this config is processed, should be other way around. Have a look at SyliusTaxationExtension.